### ISSUE TITLE
Priority should be rephrased

### ISSUE THREAD

**Issue by rdbahm** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#714]
The documentation states that "The range with the highest priority will be applied to the call." This is ambiguous, and the actual behavior seems to be that the range with the LOWEST priority (as in, 0="Highest") takes priority. It should be made more clear which way is "highest."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 205d9ebe-4e09-e500-1430-790323a47470
* Version Independent ID: 8d48e64d-34d2-fa07-dcee-f727c7ec5cad
* Content: [Create or modify an unassigned number range in Skype for Business Server](https://docs.microsoft.com/en-us/skypeforbusiness/deploy/deploy-enterprise-voice/create-or-modify-an-unassigned-number-range)
* Content Source: [Skype/SfbServer/deploy/deploy-enterprise-voice/create-or-modify-an-unassigned-number-range.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbServer/deploy/deploy-enterprise-voice/create-or-modify-an-unassigned-number-range.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @kenwith
* Microsoft Alias: **kenwith**

**Comment by AndreaBarr** [comment]
@rdbahm Thank you for this feedback

I want to make sure this quilifier if made more clear. I am sending this to the Teams team so they can properly address this. 

I was able to get this into the backlog for the Teams team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
